src/main-process/atom-application.js

```diff
@@ -93,14 +93,14 @@ class AtomApplication extends EventEmitter {
     this.quitting = false
     this.getAllWindows = this.getAllWindows.bind(this)
     this.getLastFocusedWindow = this.getLastFocusedWindow.bind(this)
-
     this.resourcePath = options.resourcePath
     this.devResourcePath = options.devResourcePath
     this.version = options.version
     this.devMode = options.devMode
     this.safeMode = options.safeMode
     this.socketPath = options.socketPath
     this.logFile = options.logFile
+    this.projectSettings = options.projectSettings
```

Comment:
I wouldn't think that we'd need to store the project settings on the entire `AtomApplication`; I would think that they would only apply to a particular window, kind of like the `pathsToOpen` options. Do we need this property?